Dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM quay.io/ukhomeofficedigital/centos-base:latest
+FROM almalinux:9.5
```

Comment:
There is a Rocky Linux image in Quay, specifically built as CentOS base image replacement.

https://quay.io/repository/ukhomeofficedigital/rockylinux-base?tab=tags

You can still use Alma if you want, so this is just a suggestion.